edge/stack/stack.go

```diff
@@ -196,6 +198,19 @@ func (manager *StackManager) Start() error {
 				ctx := context.TODO()
 
 				if stack.Action == actionDeploy || stack.Action == actionUpdate {
+					for stackImage, cacheImage := range stack.ImageMapping {
+						//pull cache image
+						//push stackImage to docker
+						// TODO (on server too): this code doesn't like image tag == "<none>", so we need to default that to ":latest"
+						// TODO: make sure the image is right for this agent's arch.
```

Comment:
is that even possible? maybe that's something that should be pre-handled from the portainer server